### PR TITLE
Update actions

### PR DESCRIPTION
**Pull request by wrt54g**
I updated the GitHub actions to later versions.

### CODE REVIEW

**Review by d-a-v — APPROVED** [review]
Thanks,
I see no objection but I still wonder what's hiding behind such changes

**Comment by mcspr** [comment]
Why not v3 for checkout?
Can't we use dependabot to avoid human-dependabot dependency?
https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

**Comment by wrt54g** [comment]
@d-a-v When will this get merged?
@mcspr This PR updates all actions/checkout to v3 already.

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -22,15 +22,15 @@ jobs:
       matrix:
         chunk: [0, 1, 2, 3, 4, 5, 6, 7]
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3.0.2
```
Comment (mcspr) [inline .github/workflows/pull-request.yml:25]: ^ this and cache. why not v3?

Comment (wrt54g) [reply .github/workflows/pull-request.yml:25]: @mcspr Because 3.0.0 is an earlier version than 3.0.2?

Comment (mcspr) [reply .github/workflows/pull-request.yml:25]: no? :)
action tags like v3 are updated during the action lifetime, they always point to the latest version available

Comment (mcspr) [reply .github/workflows/pull-request.yml:25]: https://github.com/actions/cache/tags

**Comment by wrt54g** [comment]
Suspended by #8624

**Pull request closed without merging**